pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[tool.poetry]
```

Comment:
I didn't know about poetry, looks very interesting :)

Reply:
seems to become the standard, that's why I give it a try. I currently have no strong opinion about it.